### DISCUSSION TITLE
新版本太棒了

### DISCUSSION THREAD

**Discussion by sjchenhui** [ZuodaoTech/everyone-can-use-english#1073] [Ideas]
![屏幕截图 2024-09-16 072947](https://github.com/user-attachments/assets/645b205e-25ec-4c0b-a9cb-979915c690db)
新版本的笔记直接在朗读文本处显示，能够在再次练习时直接看到，非常棒！为老师点赞

**Comment by bluechallenge** [comment]
选中一个或多个单词做笔记时，笔记会显示在第一个单词的正下方，右下方有按钮可以显示或隐藏笔记（或音标），这个功能已经支持有一段时间了。

<img width="827" alt="Note" src="https://github.com/user-attachments/assets/30e82a7f-c6b1-40d7-a216-2806007e0204">

另外，笔记功能支持[Makrdown语法](https://www.markdownguide.org)，可以设置加粗、斜体等等，不过渲染后的效果仅限于显示在笔记区的下半区（而不是单词正下方）。

![IMG_1001](https://github.com/user-attachments/assets/115e62c4-1990-43b5-a5a4-3e302c1fb248)
